.github/workflows/ci.yml

```diff
@@ -102,8 +102,10 @@ jobs:
             { os: ubuntu, version: 20.04, package-type: DEB, tag: ''},
           ]
         arch: [amd64]
+        install-agent: [true, false]
     with:
       target: ${{ matrix.target.os }}-${{ matrix.target.version }}
       arch: ${{ matrix.arch }}
       package-type: ${{ matrix.target.package-type }}
+      install-agent: ${{ matrix.install-agent }}
```

Comment:
>install-agent [](http://example.com/codeflow?start=32&length=13)

We may want to change this name as it may cause confusion. To "install-osconfig".